test/functional/feature_asmap.py

```diff
@@ -38,6 +41,10 @@ class AsmapTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
 
+    def set_peers_dat(self):
+        peers_data = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data/peers.dat')
+        shutil.copyfile(peers_data, os.path.join(self.datadir, 'peers.dat'))
```

Comment:
The file is .2 MB, so I am wondering if there is a way to generate it with a few lines of python? Can `addpeeraddress` fill the tried table?

Reply:
Didn't immediately see how to add tried entries programmatically but I think the file can be much smaller if concept ack.

Reply:
An alternative is always to add the data to https://github.com/bitcoin-core/qa-assets, which doesn't slow down the main repo clone at all.

Reply:
Reduced the `peers.dat` size 20x while still ensuring the test passes/fails

before
```
236 KB
Addrman checks started: new 3157, tried 96, total 3253
```
after
```
11.9 KB
Addrman checks started: new 95, tried 3, total 98
```

Reply:
(The new -checkaddrman option and logging is super helpful :)